scrapy/pipelines/files.py

```diff
@@ -321,6 +365,11 @@ def from_settings(cls, settings):
         gcs_store.GCS_PROJECT_ID = settings['GCS_PROJECT_ID']
         gcs_store.POLICY = settings['FILES_STORE_GCS_ACL'] or None
 
+        ftp_store = cls.STORE_SCHEMES['ftp']
+        ftp_store.FTP_USERNAME = settings['FTP_USER']           # Default is 'anonymous'
```

Comment:
:lipstick: I think it’s better to remove the comments about their default values; in case we update the default values in the future, we are will likely not remember to update them here as well.

Reply:
Will do